server/src/services/check.service.ts

```diff
@@ -0,0 +1,96 @@
+import { getRepository } from 'typeorm';
+import { CourseTask, Student, Task, TaskSolutionResult, User } from '../models';
+
+const LOW_ERROR_RATE = 0.9;
+const HIGH_ERROR_RATE = 1.1;
+const TASK_REVIEW_COUNT = 4;
+const MIN_LENGTH_MESSAGE = 70;
```

Comment:
Probably for future implementation.
But I think it will be fine to setup such variables from the UI.
What do you think?

Reply:
It'll be in future )